me.ppvan.psequel.json

```diff
@@ -0,0 +1,78 @@
+{
+    "app-id": "me.ppvan.psequel",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.vala"
+    ],
+    "command": "psequel",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/vala/bin",
+        "prepend-ld-library-path": "/usr/lib/sdk/vala/lib",
+        "env": {}
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "psequel",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/ppvan/psequel",
+                    "tag": "v0.1.1",
+                    "commit": "2a8f412d22b87dd706c6e31d0fa248fdd94a5c94"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "blueprint-compiler",
```

Comment:
there should be a cleanup up to remove it all as this s only a build dependency.

Reply:
Can you give an example?
I looked other project using blueprints and they don't do anything special

Reply:
```json
cleanup: [
    "*"
],
```

if they don't do that they do it wrong.

Reply:
thanks, i 'll commit the changes with the postgres build-options. No idea what it is yet